api/conf/conf.go

```diff
@@ -42,16 +41,50 @@ const (
 )
 
 var (
-	ENV        string
-	basePath   string
-	Schema     gjson.Result
-	DagLibPath = "/go/manager-api/dag-to-lua/"
+	ENV           string
+	basePath      string
+	Schema        gjson.Result
+	DagLibPath    = "/go/manager-api/dag-to-lua/"
```

Comment:
I don't think root dir is good

Reply:
Yep, and developers couldn't find those directories actually 🤔

![image](https://user-images.githubusercontent.com/2106987/97891590-ef63d180-1d69-11eb-930b-0e4583ff0123.png)

Reply:
it's a history issue. working on it.